### DISCUSSION TITLE
What do I put in `url` in `_config.yml`, custom domain or repository pages url?

### DISCUSSION THREAD

**Discussion by nyhryan** [cotes2020/jekyll-theme-chirpy#1989] [Q&A] [answered]
### Checklist

- [X] I have read and accepted the [contributing guidelines](https://github.com/cotes2020/jekyll-theme-chirpy/blob/master/docs/CONTRIBUTING.md).

### How did you create the site?

Generated from `chirpy-starter`

### Description

I have setup my blog as nyhryan.github.io (the default github page url), and recently added custom domain in github page setting to use ataidev.cc, following github's guide. (https://docs.github.com/en/pages/configuring-a-custom-domain-for-your-github-pages-site). nyhryan.github.io now redirects(?) to ataidev.cc.

### Operations you have already tried

```yml
# fill in the protocol & hostname for your site, e.g., 'https://username.github.io'
url: "https://nyhryan.github.io" or "https://ataidev.cc" ??
```

Which url should I put in in `url` option in `_config.yml`?

**Comment by kungfux** [comment, marked as answer]
I think it should be the actual domain of your site, which is "https://ataidev.cc".
I assume you've already tried to define it, but it's unclear what problem you had.

**Reply by nyhryan** [reply]
Oh I was just confused since it was my first time configuring custom domain for GitHub Pages. Plus I wasn't sure if everything is working correctly with Google Search engine or Bing webmaster since they take very long time to index stuffs.
